network/network-tutorial-overlay.md

```diff
@@ -274,22 +274,21 @@ This tutorial assumes the swarm is already set up and you are on a manager.
 
 ## Use an overlay network for standalone containers
 
-This example does the following:
+This example demonstrates how to communicate between standalone containers on
+different Docker daemons using an overlay network. Steps are:
 
-- initializes a swarm on `host1`
-- joins `host2` to the swarm
-- creates an attachable overlay network
-- creates an `alpine` service with 3 replicas, connected to the overlay network
-- creates a single `alpine` container on `host2`, which is also attached to the
-  overlay network
-- proves that the standalone container can communicate with the service tasks,
-  and vice versa.
+- On `host1`, initialize the node as a swarm (manager)
```

Comment:
I know this is not done consistently across the docs but we end all bullets with a final stop.